src/shared/services/i18n/fallback/locales/nl/translations.json

```diff
@@ -0,0 +1,63 @@
+{
+    "algemeen": {
+        "toestemming-delen-contactgegevens": "Ja, ik geef Signalen toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
+        "wijk": "Wijk",
+        "kop-titel": "",
+        "helptekst": "Wij zijn bereikbaar van maandag tot en met vrijdag van 08.00 tot 18.00 uur.",
+        "helptekst-kop": "Lukt het niet om een melding te doen? Bel het telefoonnummer [070-373 8393](tel:+31703738393)",
+        "logo-omschrijving": "Logo gemeente",
```

Comment:
Do we want the keys to be kebab-case? Might be more difficult to work with in a js context, compared to camelCase for example